Improve GitHub bug report issue template

janpio opened the pull request:
<!-- Thanks for contributing to _fastlane_! Before you submit your pull request, please make sure to check the following boxes by putting an x in the [ ] (don't: [x ], [ x], do: [x]) -->

### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context
While triaging the issues the last few weeks, I noticed a few recurring problems: the command being executed is missing, people apologize that `--capture_output` didn't work with their command - and many people still ignore the issue template completely even when it would be _very_ useful for them to fill it.

### Description
I hope some of these changes will help a bit:
- Explicitly ask for command that was executed
- Include that `--capture_output` doesn't work for all commands
- Remove Emoji that isn't rendered in the comment anyway
- Reformat multi line comments
- Replace `[INSERT OUTPUT HERE]` with `[REPLACE THIS WITH YOUR INFORMATION]` in the hope more people will actually do this

janpio committed:
Improve bug template

- Explicitly ask for command that was executed
- Include that `--capture_output` doesn't work for all commands
- Remove Emoji that isn't rendered in the comment anyway
- Reformat multi line comments
- Replace `[INSERT OUTPUT HERE]` with `[REPLACE THIS WITH YOUR INFORMATION]` in the hope more people will actually do this

fastlane-bot-helper commented:
<!--
  0 Errors
  1 Warning: Pull Request is Work in Progre...
  0 Messages
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Pull Request is Work in Progress</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

janpio committed:
update checklist instructions

as [ x] and similar are covered by https://github.com/fastlane/issue-bot/pull/50

revolter commented on .github/ISSUE_TEMPLATE/BUG_REPORT.md:
```diff
@@ -16,20 +16,26 @@ about: If something isn't working as expected 🤔
-##### Complete output when running fastlane, including the stack trace and command used
-<!-- You can use: `--capture_output` as the last commandline argument to get that collected for you -->
+##### Command executed
+<!-- The command you executed on the command line, that resulted in an error -->
```
You could remove the comma.

janpio replied:
```suggestion
<!-- The command you executed on the command line that resulted in an error -->
```

revolter commented on .github/ISSUE_TEMPLATE/BUG_REPORT.md:
```diff
@@ -16,20 +16,26 @@ about: If something isn't working as expected 🤔
-<!-- The output of `--capture_output` could contain sensitive data such as application ids, certificate ids, or email addresses, Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue -->
+##### Complete output when running fastlane, including the stack trace and command used
+<!-- 
+You can use `--capture_output` as the last commandline argument for many commands to get that collected for you. Otherwise, please do it manually.
```
Space inside `commandline`.

janpio replied:
```suggestion
You can use `--capture_output` as the last command line argument for many commands to get that collected for you. Otherwise, please do it manually.
```

revolter commented on .github/ISSUE_TEMPLATE/BUG_REPORT.md:
```diff
@@ -16,20 +16,26 @@ about: If something isn't working as expected 🤔
-If you used `--capture_output` option, please remove this block as it is already included there. -->
+<!-- 
+Please run `fastlane env` and copy the output below. This will help us help you.
+If you used `--capture_output` option, please remove this block as it is already included there. 
```
`used the`?

janpio replied:
```suggestion
If you used the `--capture_output` option, please remove this block as it is already included there. 
```

revolter reviewed (dismissed):
Other than these, the changes are really great! 👍

janpio committed:
Apply suggestions from @revolter 's code review
